### ISSUE TITLE
specparam - FOOOF confusion

### ISSUE THREAD

**Issue by chapochn** [fooof-tools/fooof#318]
Hello,

in the quickstart example it's written:

`from specparam import SpectralModel`

But I don't see the option to download specparam2.0 anywhere. Am I missing something?

Also running FOOOF I am getting:
```
The `fooof` package is being deprecated and replaced by the `specparam` (spectral parameterization) package.
This version of `fooof` (1.1) is fully functional, but will not be further updated.
New projects are recommended to update to using `specparam` (see Changelog for details)
```

I am not seeing how to start using specparam. Could you please advise? Thank you!

**Comment by TomDonoghue** [comment]
Sorry for the confusion - `specparam` hasn't been properly released yet, so fooof 1.1 is the most recent fully available version. We added the note about updating because _soon_ there will be a new release that effectively deprecates the fooof package, but there's still a bit more work to do before everything officially switches, so until then the Github is basically ahead of what's released, and the warning on 1.1 is a bit premature. 

I'll look into tweaking the docs and/or warning to be clearer on this.

**Comment by chapochn** [comment]
ok, gotcha, thanks for clarifying!
